cdap-ui/app/cdap/components/CaskWizards/AddNamespace/index.js

```diff
@@ -51,7 +55,27 @@ export default class AddNamespaceWizard extends Component {
       type: AddNamespaceActions.onReset
     });
   }
-
+  buildSuccessInfo(responseText) {
+    // responseText has the format "Namespace '{namespace}' created successfully.""
```

Comment:
extra quote at the end.

Should open an improvement backend JIRA for this as well